src/Draco.Compiler.Tests/Syntax/ParserTests.cs

```diff
@@ -1802,4 +1806,15 @@ public void TestEmptyModuleDeclarationStatement()
             }
         }
     }
+
+    [Fact]
+    public void TestImportDeclarationWithVisibilityModifier()
+    {
+        this.ParseDeclaration("public import Foo;");
+
+        this.N<UnexpectedDeclarationSyntax>();
+        {
+            this.InvalidT(TokenKind.KeywordPublic);
+        }
```

Comment:
There is no helper for parsing multiple declaration, so I didn't tested that the rests of the tokens are present.